app/v1/connectors/RetrieveBusinessDetailsConnector.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.connectors
+
+import config.AppConfig
+import javax.inject.Inject
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+import v1.connectors.httpparsers.StandardDesHttpParser._
+import v1.models.request.retrieveBusinessDetails.RetrieveBusinessDetailsRequest
+import v1.models.response.retrieveBusinessDetails.RetrieveBusinessDetailsResponseList
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class RetrieveBusinessDetailsConnector @Inject()(val http: HttpClient,
+                                                 val appConfig: AppConfig) extends BaseDesConnector {
+def retrieveBusinessDetails(request: RetrieveBusinessDetailsRequest)
+                           (implicit hc: HeaderCarrier, ec: ExecutionContext):
+                            Future[DesOutcome[RetrieveBusinessDetailsResponseList]] = {
+
+    val url = s"registration/business-details/nino/${request.nino}"
+  get(
+    DesUri[RetrieveBusinessDetailsResponseList](s"$url")
```

Comment:
From the Slack conversation we had: 

Assuming DES returns
```
[
  {
    "field1": 123
  },
  {
    "field1": 456
  }
]
```
and we want just
```
{
  "field1": 123
}
```
then you can just write `DesUri[Seq[RetrieveBusinessDetailsResponse]]`; putting it in another case class will make Play look for
```
{
  "retrieveBusinessDetailsResponse": [
    {
      "field1": 123
    },
    {  
      "field1": 456
    }
  ]
}
```
which DES doesn’t return so it won’t work